beacon_node/src/cli.rs

```diff
@@ -87,148 +89,147 @@ pub fn cli_app<'a, 'b>() -> App<'a, 'b> {
                       - --listen-address '0.0.0.0' --listen-address '::' will listen over both \
                       IPv4 and IPv6. The order of the given addresses is not relevant. However, \
                       multiple IPv4, or multiple IPv6 addresses will not be accepted.")
-                .multiple(true)
-                .max_values(2)
+                .action(ArgAction::Append)
+                // .max_values(2)
```

Comment:
Can probably remove this comment. There isn't a way to enforce 2 values? I guess we can check at validation time.

Reply:
You have to use
```
                .num_args(0..=2)
```